zipline/lib/adjusted_array.pyx

```diff
@@ -216,10 +227,17 @@ cdef class _Float64AdjustedArrayWindow:
         self.anchor += 1
         return out
 
-    def __repr__(self):
-        return "%s(window_length=%d, anchor=%d, max_anchor=%d)" % (
-            type(self).__name__,
-            self.window_length,
-            self.anchor,
-            self.max_anchor,
+    def inspect(self):
```

Comment:
I had it as the repr originally, but that's really annoying if you have a list of these or something.